src/wsproxy.ml

```diff
@@ -45,7 +46,15 @@ let start path handler =
          Lwt_unix.close fd_sock2 >>= fun _ ->
```

Comment:
This is started on demand by the server, no systemd involved, and uses unix domain sockets and not inet to proxy between the websockets connection and the local vnc